packages/devkit/src/utils/module-federation/dependencies.ts

```diff
@@ -53,7 +53,7 @@ function collectDependencies(
   return dependencies;
 }
 
-function getLibraryImportPath(
+export function getLibraryImportPath(
```

Comment:
Let's not export this - I don't think devkit's public api should change for this.

Reply:
Will remove it from `public-api.ts`, yet need to keep the export for internal usage